newIDE/app/src/GameDashboard/PublicGamePropertiesDialog.js

```diff
@@ -247,17 +244,13 @@ export const PublicGamePropertiesDialog = ({
         disabled={isLoading}
         canBePublishedOnGdGames={canBePublishedOnGdGames}
       />
-      {gameUnregisterErrorText && (
```

Comment:
removed this to make it an alert, allowing to re-use the same function to register in multiple places without having to handle where to display that errorText in each case